src/index.js

```diff
@@ -1,19 +1,57 @@
 // 1. Write a function that accepts an array of numbers. Return a new array with all elements doubled
 // Ex: double([1,2,3]) => [2,4,6]
+function doubleAll(element){
```

Comment:
I think the function is meant to be able to directly take any array as argument.